defaults.js

```diff
@@ -21,21 +23,68 @@ export async function applyDefaults () {
       }
     }
 
-    // Apply theme if available
-    if (defaults.theme) {
-      applyTheme(defaults.theme)
+    // Determine if the custom theme differs from the default light theme
```

Comment:
The "custom" option will appear as a default option if the social reader default json list changes.